corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -136,7 +136,7 @@
         {% endcompress %} 
 
         {% if formdesigner %}
-        <script src="{% new_static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
+        <script src="{% new_static 'jquery-migrate/jquery-migrate.min.js' %}"></script>
```

Comment:
fyi this can be removed. I removed it https://github.com/dimagi/commcare-hq/commit/8d89839670e3ba414c74d98a9d3792bcd244b644 but it's in the b3 app manager branch